lib/config/configuration.go

```diff
@@ -398,9 +399,7 @@ func applyLogConfig(loggerConfig Log, logger *log.Logger) error {
 		logger.SetOutput(logFile)
 	}
 	switch strings.ToLower(loggerConfig.Severity) {
-	case "":
-		break // not set
-	case "info":
+	case "", "info":
```

Comment:
`info` makes sense as the default log level.
without this, if `severity` is not specified in the config, the standard logger ends up at `error` level and misses lots of useful entries